app/models/export.rb

```diff
@@ -10,7 +10,11 @@ def initialize(account)
   end
 
   def to_blocked_accounts_csv
-    to_csv account.blocking.select(:username, :domain)
+    CSV.generate(headers: ['Account address', 'Stealth'], write_headers: true) do |csv|
```

Comment:
Usage of the word "stealth" in end-user oriented strings is probably worth a discussion

Same for "account address" to be honest. Do we use that anywhere else? Do we use "handle" or "username" in other places? It should be consistent.

Reply:
Well, “Account address” is already used as a CSV header for other types of lists… as for “Stealth”, I'm open to suggestions.